src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -88,10 +88,10 @@ const WeeklyCommittedHours = props => {
   return (
     <Input
       type="number"
-      name="weeklycommittedHours"
-      id="weeklycommittedHours"
-      data-testid="weeklycommittedHours"
-      value={props.userProfile.weeklycommittedHours}
```

Comment:
Please, make it back to `weeklycommittedHours` 
as you can see it's not showing the correct value:
![image](https://user-images.githubusercontent.com/17365161/225967931-bf1ce4e7-2f38-4ea1-a48a-200634abf64a.png)